integration_test/cases/migrator.exs

```diff
@@ -81,6 +86,9 @@ defmodule Ecto.Integration.MigratorTest do
 
       assert count_entries() == 1
       assert [50] = run(TestRepo, path, :up, to: 50, log: false)
+
+      purge migrations
```

Comment:
I think it's because migrator will `Code.load_file`: https://github.com/elixir-lang/ecto/blob/master/lib/ecto/migrator.ex#L213